src/ai/chatAi.ts

```diff
@@ -46,6 +52,62 @@ export default class ChatAi {
     return this.ain.sendTransaction(txBody);
   }
 
+  async depositCredit(
+    provider: string,
+    api: string,
+    amount: number
+  ): Promise<CreditDepositTransactionResult> {
+    const address = this.ain.signer.getAddress();
+
+    const aiName = validateAndGetAiName(provider, api);
+    const aiService = await validateAndGetAiService(aiName, this.ainize);
+
+    // TODO(jiyoung): update login method to use signer. (if implemented)
+    await AinizeAuth.getInstance().login(this.ain, this.ainize);
+    const currentBalance = await aiService.getCreditBalance();
+    const txHash = await aiService.chargeCredit(amount);
+    const updatedBalance = await this.waitForBalanceUpdate(
```

Comment:
@platfowner @minsulee2 After charge credits, I added a `waitForBalanceUpdate()` method to wait up to 1 minute and check every 5 seconds if the balance is updated. (In test, it took about 5.xx seconds)